src/app/sportovni-akce/[category]/[id]/page.tsx

```diff
@@ -45,14 +72,45 @@ const EventPage: NextPage<EventPageProps> = async ({ params }) => {
             <h1 className="mt-12 text-center text-2xl font-medium leading-normal md:mt-9 lg:text-start lg:text-4xl">
               {name}
             </h1>
-            <p
-              className="mt-8 text-lg font-light"
-              dangerouslySetInnerHTML={{ __html: description }}
+            <p className="mt-8 text-lg font-light">{description}</p>
+            <EventDetail
+              className="mx-auto mt-8 lg:hidden"
+              event={event}
+              formattedDateTime={{ formattedDate, formattedTime }}
             />
-            <EventDetail className="mx-auto mt-8 lg:hidden" event={event} />
             <hr className="my-16 border-t border-low-contrast" />
           </section>
+
+          <section className="w-full">
+            <h2 className="flex flex-col px-20 text-center text-xl font-medium leading-normal md:flex-row md:px-0 lg:text-start lg:text-3xl">
+              Chceš se zúčastnit? Registruj se na akci u instruktora!
```

Comment:
I would go with different text. I can make the event, you can make the event. We are hardly instructors